### ISSUE TITLE
TextArea and TextField Default Values not available when Conditional Logic in place

### ISSUE THREAD

**Issue by du8die** [formio/formio.js#1278]
Good afternoon,

In the form builder:

Both the TextArea and TextField components (and I'm assuming any other components that derive from them) have an issue where any time conditional view logic is in place, the Default Value options on the "Data" tab in the editor form are not visible.

Thanks!

**Comment by wag110894** [comment]
Hello, 
We are aware of this issue and have this item on one of our developers list as a high priority. We will let you know once this item is resolved. 

Thanks for reaching out!

**Comment by du8die** [comment]
Do you happen to have an ETA on this fix?

Thanks!

**Comment by DavidPLamas** [comment]
Hi,
Any progress on this ?

**Comment by wag110894** [comment]
Thank you for your patience.
This issue has been resolved on the latest version of formio.js (3.23.0).

**Comment by sriram275** [comment]
Hi,

I am still able to see this issue with formiojs version **3.29.9**

Not working with below:
```json
{
    type: 'textarea',
    key: 'custom',
    label: 'Button Custom Logic',
    tooltip: 'The custom logic to evaluate when the button is clicked.',
    rows: 5,
    editor: 'ace',
    input: true,
    weight: 120,
    placeholder: "data['mykey'] = data['anotherKey'];",
    defaultValue: "data['mykey'] = 'testValue'",
    conditional: {
      json: { '===': [{ var: 'data.action' }, 'custom'] }
    }
  }
```

And working code:
```json
{
    type: 'textarea',
    key: 'custom',
    label: 'Button Custom Logic',
    tooltip: 'The custom logic to evaluate when the button is clicked.',
    rows: 5,
    editor: 'ace',
    input: true,
    weight: 120,
    placeholder: "data['mykey'] = data['anotherKey'];",
    defaultValue: "data['mykey'] = 'testValue'"
  }
```

Only difference is **conditional tag** not present in the working code.

I was checking in Changelog to see this issue fixed. I could see 3.22.6 version changes but after that no version till  3.29.16 has captured in changelog doc.

Is this an issue 3.29.x? 
It is fixed in formiojs 3.29
